lnxrouter

```diff
@@ -101,15 +101,16 @@ Options:
                             (defaults to /etc/hostapd/hostapd.accept)
     --hostapd-debug <level> 1 or 2. Passes -d or -dd to hostapd
     --isolate-clients       Disable wifi communication between clients
+    --idle-timeout <time>   Idle timeout (in seconds) to disconnect a client device
     --no-haveged            Do not run haveged automatically when needed
-    --hs20                  Enable Hotspot 2.0
+    --hs20                  (ß) Enable Hotspot 2.0
```

Comment:
Have you seen any modern Linux's hostapd which doesn't support hs20? It was added in 2013 to  hostapd. I guess it has been well supported. (My distro's hostapd doesn't support wifi6 at all, but it supports hs20)